akka-bench-jmh/src/main/scala/akka/http/HttpBenchmark.scala

```diff
@@ -53,7 +53,7 @@ class HttpBenchmark {
   }
 
   @TearDown
-  def shutdown() = {
+  def shutdown() {
```

Comment:
Procedure syntax has been deprecated so it would be better to add a Unit return type.